robolectric/src/test/java/org/mockito/configuration/MockitoConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package org.mockito.configuration;
+
+public class MockitoConfiguration extends DefaultMockitoConfiguration {
+
+    @Override
+    public boolean enableClassCache() {
+        return false;
```

Comment:
Actually, the problem only arises if one uses `MultiApiRobolectricTestRunner`. I wonder how this cache works.